### PR TITLE
OSDOCS-1886: Updating the Welcome page for 4.8

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-1886

Preview build: https://deploy-preview-33427--osdocs.netlify.app/openshift-enterprise/latest/welcome/index.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
Metering bumps are covered in https://github.com/openshift/openshift-docs/pull/33768

File: welcome/index.adoc [hunk 1]
```diff
@@ -4,33 +4,33 @@
...
 
-- **xref:../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Install a cluster on AWS]**: You have the most installation options when you deploy a cluster on Amazon Web Services (AWS). You can deploy clusters with xref:../installing/installing_aws/installing-aws-default.adoc#installing-aws-default[default settings] or xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[custom AWS settings].
+- **xref:../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Install a cluster on AWS]**: You have tmany installation options when you deploy a cluster on Amazon Web Services (AWS). You can deploy clusters with xref:../installing/installing_aws/installing-aws-default.adoc#installing-aws-default[default settings] or xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[custom AWS settings].
```
Comment (adellape) [inline welcome/index.adoc:33]: s/tmany/many/

File: welcome/index.adoc [hunk 2]
```diff
@@ -89,26 +95,21 @@ Internet access is still required to access the cloud APIs and installation medi
...
-to visually interact with your applications, monitor
-status, connect and group components,
-and modify your code base.
+to see your applications, monitor status, connect and group components, and modify your code base.
 
 - ** xref:../cli_reference/developer_cli_odo/understanding-odo.adoc#understanding-odo[Use the developer CLI tool (odo)]**:
-The odo CLI tool lets developers create single or multi-component applications
-easily and automates deployment, build, and service route configurations.
-It abstracts complex Kubernetes and {product-title} concepts, allowing developers to focus on developing their applications.
+The odo CLI tool lets developers create single or multi-component applications easily and automates deployment, build, and service route configurations. It abstracts complex Kubernetes and {product-title} concepts, allowing you to focus on developing your applications.
```
Comment (adellape) [inline welcome/index.adoc:112]: Markup `odo`?

File: welcome/index.adoc [hunk 3]
```diff
@@ -121,35 +122,31 @@ application packages on Kubernetes. A Helm chart is a packaging format that
...
 
 - **xref:../applications/deployments/what-deployments-are.adoc#what-deployments-are[Create deployments]**:  Use `Deployment` and `DeploymentConfig` objects to exert fine-grained management over applications.
-Use the Workloads page or `oc` CLI to xref:../applications/deployments/managing-deployment-processes.adoc#deployment-operations[manage deployments]. Learn xref:../applications/deployments/deployment-strategies.adoc#deployment-strategies[rolling, recreate, and custom] deployment strategies.
+ xref:../applications/deployments/managing-deployment-processes.adoc#deployment-operations[Manage deployments] using the Workloads page or OpenShift CLI (`oc`). Learn xref:../applications/deployments/deployment-strategies.adoc#deployment-strategies[rolling, recreate, and custom] deployment strategies.
```
Comment (adellape) [inline welcome/index.adoc:130]: Markup **Workloads**?

File: welcome/index.adoc [hunk 4]
```diff
@@ -200,31 +189,25 @@ xref:../storage/persistent_storage/persistent-storage-nfs.adoc#persistent-storag
...
 
 === Change cluster components
 
-- **xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[Use custom resource definitions (CRDs) to modify the cluster]**: Cluster features that are implemented with Operators, can be modified with CRDs. Learn to xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-creating-custom-resources-definition_crd-extending-api-with-crds[create a CRD] and xref:../operators/understanding/crds/crd-managing-resources-from-crds.adoc#crd-managing-resources-from-crds[manage resources from CRDs].
+- **xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[Use custom resource definitions (CRDs) to modify the cluster]**: Cluster features implemented with Operators, can be modified with CRDs. Learn to xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-creating-custom-resources-definition_crd-extending-api-with-crds[create a CRD] and xref:../operators/understanding/crds/crd-managing-resources-from-crds.adoc#crd-managing-resources-from-crds[manage resources from CRDs].
```
Comment (adellape) [inline welcome/index.adoc:199]: s/Operators, can/Operators can/

File: welcome/index.adoc [hunk 5]
```diff
@@ -200,31 +189,25 @@ xref:../storage/persistent_storage/persistent-storage-nfs.adoc#persistent-storag
...
 
 - **xref:../scalability_and_performance/scaling-cluster-monitoring-operator.adoc#scaling-cluster-monitoring-operator[Scale] and xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[tune] clusters**: Set cluster limits, tune nodes, scale cluster monitoring, and optimize networking, storage, and routes for your environment.
 
 - **Update a cluster**:
-To upgrade your {product-title} to a later version, use the Cluster Version Operator (CVO).
-If an update is available from the Container Platform update service, you apply that cluster update from either the xref:../updating/updating-cluster.adoc#updating-cluster[web console] or the xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[CLI].
+Use the Cluster Version Operator (CVO) to upgrade your {product-title} cluster. If an update is available from the Container Platform update service, you apply that cluster update from either the {product-title} xref:../updating/updating-cluster.adoc#updating-cluster[web console] or the xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[OpenShift CLI] (`oc`).
```
Comment (adellape) [inline welcome/index.adoc:210]: Do we know if "Container Platform update service" could just be "{product-title} update service" here? I see a hit for "OpenShift Container Platform update service" in `updating/installing-update-service.adoc`.

File: welcome/index.adoc [hunk 6]
```diff
@@ -239,10 +222,9 @@ xref:../updating/updating-disconnected-cluster.adoc#updating-disconnected-cluste
 
 - **xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Monitor clusters]**:
 Learn to  xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack].
-Once your monitoring is configured, use the Web UI to access xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[monitoring dashboards].
-In addition to infrastructure metrics, you can also scrape and view metrics for your own services.
+After configuring monitoring, use the Web UI to access xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[monitoring dashboards]. In addition to infrastructure metrics, you can also scrape and view metrics for your own services.
```
Comment (adellape) [inline welcome/index.adoc:225]: Can "Web UI" be "web console" in this context?

File: welcome/index.adoc [hunk 7]
```diff
@@ -62,11 +68,11 @@ user-provisioned infrastructure on
 xref:../installing/installing_aws/installing-restricted-networks-aws.adoc#installing-restricted-networks-aws[AWS],
 GCP,
```
Comment (adellape) [inline welcome/index.adoc:69]: Can "GCP" here link to https://deploy-preview-33427--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.html?

File: welcome/index.adoc [hunk 8]
```diff
@@ -89,26 +95,21 @@ Internet access is still required to access the cloud APIs and installation medi
...
 
-- **xref:../applications/projects/working-with-projects.adoc#working-with-projects[Work with projects]**: Create projects from the web console or CLI to organize and share the software you develop.
+- **xref:../applications/projects/working-with-projects.adoc#working-with-projects[Work with projects]**: Create projects from the {product-title} web console or OpenShift CLI (`oc`) to organize and share the software you develop.
 
 - **xref:../applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Work with applications]**:
-Use the xref:../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[Developer perspective] in the {product-title} web console to easily
+Use the xref:../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[Developer perspective] in the {product-title} web console to
```
Comment (adellape) [inline welcome/index.adoc:105]: Looks like in the target page, it's marked up throughout as:

```
*Developer* perspective
```

File: welcome/index.adoc [hunk 9]
```diff
@@ -89,26 +95,21 @@ Internet access is still required to access the cloud APIs and installation medi
...
 xref:../applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[create and deploy applications].
 Use the
 xref:../applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc#odc-viewing-application-composition-using-topology-view[Topology view]
```
Comment (adellape) [inline welcome/index.adoc:108]: Same here for marking up as:

```
*Topology* view
```

File: welcome/index.adoc [hunk 10]
```diff
@@ -89,26 +95,21 @@ Internet access is still required to access the cloud APIs and installation medi
...
-to visually interact with your applications, monitor
-status, connect and group components,
-and modify your code base.
+to see your applications, monitor status, connect and group components, and modify your code base.
 
 - ** xref:../cli_reference/developer_cli_odo/understanding-odo.adoc#understanding-odo[Use the developer CLI tool (odo)]**:
```
Comment (adellape) [inline welcome/index.adoc:111]: Maybe markup this `odo` in the link text too?

File: welcome/index.adoc (hunk 5 above)
Comment (adellape) [reply welcome/index.adoc:210]: Actually, in https://deploy-preview-33427--osdocs.netlify.app/openshift-enterprise/latest/updating/installing-update-service.html (linked in the next bullet) it looks like it refers to "OpenShift Update Service (OSUS)" throughout there, so if these are referring to the same thing, sync up w/ that?

**Review by adellape — COMMENTED** [review]
Excellent overall improvements (in addition to the new stuff)!

**Pull request merged**